protocol/migrations/sqlite/1733428521_pinned_messages_add_on_delete_clause.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE pin_messages_new (
```

Comment:
Please call it `_v2` to avoid `_new_new` in future 😄 
```suggestion
CREATE TABLE pin_messages_v2 (
```

Reply:
oh never mind, you delete it immediately 🤦